src/templates/program-page.js

```diff
@@ -833,17 +813,12 @@ export const query = graphql`query ($id: String) {
         }
         relationships {
           field_media_image {
-            localFile {
-              publicURL
```

Comment:
It looks like publicURL is still being used in a variable up above (line 274). We should either update the variable to remove publicURL or update the query

Reply:
Good catch - it's used for the Twitter/Facebook cards. Restored.